base/array.jl

```diff
@@ -956,6 +956,14 @@ function intersect(v1, vs...)
     ret
 end
 
+"""
+    isdisjoint(v1, v2) -> Bool
+
+Determine whether the collections `v1` and `v2` are disjoint, using
+the function `in` to check.
+"""
+isdisjoint(v1, v2) = all(v -> v ∉ v1, v2)
```

Comment:
Probably iterate over the smaller of the two sets?  If one of the sets is much bigger than the other that could make a huge difference.

Also if the point of putting it in the standard library is efficiency, probably an explicit loop would be better than a higher-order function.